node/node.go

```diff
@@ -560,7 +572,7 @@ func (node *Node) Start(ctx context.Context) error {
 			// See https://github.com/filecoin-project/go-filecoin/issues/1105
 			chainSynced.Done()
 		}
-		node.HelloSvc = hello.New(node.Host(), node.ChainReader.GenesisCid(), syncCallBack, node.PorcelainAPI.ChainHead, node.Repo.Config().Net, flags.Commit)
+		node.HelloSvc = hello.New(node.Host(), node.ChainReader.GenesisCid(), node.PeerTracker, syncCallBack, node.PorcelainAPI.ChainHead, node.Repo.Config().Net, flags.Commit)
```

Comment:
Nearly-blocking: instead of coupling hello with the tracker, add a peer to the tracker in the existing syncCallBack. 
- Perhaps change the callback to use the new ChainInfo type
- Perhaps rename that callback to `helloCallback` so that hello is less prescriptive.